code/go/ecs/process.go

```diff
@@ -31,9 +31,6 @@ type Process struct {
 	// Process id.
 	PID int64 `ecs:"pid"`
 
-	// Process id.
-	ParentPID int64 `ecs:"parent.pid"`
```

Comment:
Should `Process` be self-referential?
```
      *Process parent `ecs:"process.parent"`
```

Reply:
This Golang library is going away from this repo in the future. Does the Endpoint team use it?

Reply:
nope, just wondering